spec/system/provider_interface/rejection_reasons_spec.rb

```diff
@@ -33,6 +33,24 @@
     then_i_can_see_the_rejected_application_feedback
   end
 
+  scenario 'giving feedback on RBD application using redesigned rejection form', with_audited: true do
```

Comment:
I think we moved away from having multiple scenarios per system spec? Although we could reuse a lot of the steps I reckon we can move this to its own spec to avoid all the branching, what do you think?

Reply:
Yeah it felt a bit wrong doing this, my thinking was that we would move these scenarios out into the relevant system specs once we release this feature and remove the old form/wizard/behaviour.